src/metabase/models/dashboard_tab.clj

```diff
@@ -102,3 +113,48 @@
   (when (seq tab-ids)
     (t2/delete! :model/DashboardTab :id [:in tab-ids]))
   nil)
+
+(mu/defn classify-changes :- [:map
```

Comment:
not sure where to put this tbh, we can put it in `metabase.util` but it's currently written in `cljc` and it doesn't support malli schema :/

Reply:
Can't you put it in a reader macro inside `metabase.util`?